.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
I think this entire `.idea` directory should not be in the repo; maybe add to `.gitignore`?

Reply:
why to these sneak in!! come on!!